app/src/main/java/org/simple/clinic/setup/SetupActivity.kt

```diff
@@ -106,7 +105,7 @@ class SetupActivity : AppCompatActivity(), UiActions {
     val wrappedContext = baseContext
```

Comment:
We are doing this everywhere, should we extract this into a separate function? Thoughts?

Reply:
We should, I'll pick this up after finishing the feature.